client/whiteboard/socket/useSocket.ts

```diff
@@ -1,13 +1,15 @@
 import { Dispatch, MutableRefObject, useEffect, useRef } from "react";
 import { BoardAction } from "../state/action";
+import getConfig from "next/config";
 
 type Updater = (update: BoardAction) => void
 
 export const useSocket = (id: string, dispatch: Dispatch<BoardAction>): Updater => {
+    const { publicRuntimeConfig } = getConfig();
```

Comment:
this is a [react hook](https://reactjs.org/docs/hooks-intro.html) - and thus will be executed multiple times during the component lifecycle. stuff like this is best setup outside of hooks so it is only executed once (this can cause interesting perf issues in the client). not a deal breaker for this PR, but we will want to lift this to be "global" to the file (node module)

Reply:
Ok, I didn't know that. Do you want me to fix this before merging? Or is it fine to get back here after merge?